src/components/Wizard/PWizard.vue

```diff
@@ -36,6 +36,7 @@
           </p-button>
         </slot>
       </div>
+      <slot name="footer" />
```

Comment:
You could use the same logic from the content slot and have the footer be dynamic based on the step. That way you could target the footer for a specific step.